src/backend/distributed/executor/local_executor.c

```diff
@@ -373,11 +372,11 @@ ExtractParametersForLocalExecution(ParamListInfo paramListInfo, Oid **parameterT
 
 
 /*
- * LocallyExecuteUtilityTask executes the given local task query in the current
+ * ExecuteUtilityCommand executes the given local task query in the current
  * session.
  */
-static void
-LocallyExecuteUtilityTask(const char *localTaskQueryCommand)
+void
+ExecuteUtilityCommand(const char *localTaskQueryCommand)
```

Comment:
This previously looked like a local execution specific function, but with the rename it doesn't seem anymore. Its implementation seems to work for non-local commands too, but it needs some clean-up (in names & comments) to remove the "local" references to avoid confusion.

Since @onderkalaci implemented the local executor, can you double check with him to see if we haven't missed anything?

Reply:
That's right, the input can be any utility command. So I think we should follow Hadi's comment.